docs/book/v5/tutorials/find-user-by-identity.md

```diff
@@ -0,0 +1,213 @@
+# Find user by identity - A practical example
+
+## Our goal
+
+Create a new endpoint that fetches a user based on its identity.
+
+We already have an endpoint that retrieves a user based on their UUID, so we can review it and create something similar.
+
+## What we have
+
+Let's print out all available endpoints using :
+
+```shell
+php ./bin/cli.php route:list
+```
+
+This command will list all available endpoints, which looks like this:
+
+```text
++--------+---------------------------------+--------------------------------+
+| Method | Name                            | Path                           |
++--------+---------------------------------+--------------------------------+
+| POST   | account.activate.request        | /account/activate              |
+| PATCH  | account.activate                | /account/activate/{hash}       |
+| PATCH  | account.modify-password         | /account/reset-password/{hash} |
+.............................................................................
+.............................................................................
+.............................................................................
+| GET    | user.my-avatar.view             | /user/my-avatar                |
+| GET    | user.role.list                  | /user/role                     |
+| GET    | user.role.view                  | /user/role/{uuid}              |
+| PATCH  | user.update                     | /user/{uuid}                   |
+| GET    | user.view                       | /user/{uuid}                   |
++--------+---------------------------------+--------------------------------+
+```
+
+### Note
+
+> **The above output is just an example.**
+> 
+> More info about listing available endpoints can be found [here](../commands/display-available-endpoints).
+
+The endpoint we're focusing on is the last one, `user.view`, so let's take a closer look at its functionality.
+
+If we search for the route name `user.view` we will find its definition in the `src/User/src/RoutesDelegator.php` class, here live all user related endpoints.

```

Comment:
instead of `here live all user related endpoints.`
`where all user related endpoints are found.`